sw/applications/example_power_gating_core/example_power_gating_core.c

```diff
@@ -52,7 +50,7 @@ int main(int argc, char *argv[])
     }
 
     // Power gate the core and wait for rv_timer interrupt
-    if (power_gate_core(&power_manager, kTimer, &power_manager_cpu_counters) != kPowerManagerOk)
+    if (power_gate_core(&power_manager, kTimer_0, &power_manager_cpu_counters) != kPowerManagerOk)
```

Comment:
please test the applications with all the 4 timers and the DMA, also, shall we add the SPI as an event? if so, please test it

Reply:
I discussed with Ruben and he implemented the system in such a  way to generate a single interrupt from the DMA when the complete acquisition (the includes multiple SPI transactions) is completed. So, I would not add the SPI as a wake-up event, for the moment, but we can discuss.

Reply:
and you do not use the DMA because the DMA is busy on doing something else?

Reply:
I added the SPI interrupt as well.

Reply:
Solved.